src/unit_tests/dataset_tests.cpp

```diff
@@ -114,9 +114,11 @@ BOOST_AUTO_TEST_CASE(get_offset)
     // - https://github.com/geotrellis/gdal-warp-bindings/pull/96 (PR that introduced this comment)
     // - https://github.com/OSGeo/gdal/issues/2579
     // - https://github.com/OSGeo/gdal/commit/69f25f253d141faf836c400676f9f94dd3f43707
-    // 
+    //
     // BOOST_TEST(success != false);
+#if !defined(PRE_GDAL_31)
     BOOST_TEST(success == false);
+#endif
```

Comment:
Hmm, there is GDAL 3.2 in _unstable_ packages (see: https://packages.debian.org/search?keywords=gdal) would it be possible just to use GDAL 3.2?